sse2neon.h

```diff
@@ -8946,6 +8952,38 @@ FORCE_INLINE int _sse2neon_sido_negative(int res, int lb, int imm8, int bound)
     int r2 = (_sse2neon_cmpfunc_table[imm8 & 0x0f])(a, la, b, lb); \
     r2 = _sse2neon_sido_negative(r2, lb, imm8, bound)
 
+#define SSE2NEON_CMPSTR_GENERATE_INDEX(r2, bound, imm8)          \
+    return (r2 == 0) ? bound                                     \
+                     : ((imm8 & 0x40) ? (31 - __builtin_clz(r2)) \
```

Comment:
For the sake of potential MSVC support, you shall use a wrapper function. That is, declare a static inline `_sse2neon_clz` with MSVC implementation. See https://stackoverflow.com/questions/355967/how-to-use-msvc-intrinsics-to-get-the-equivalent-of-this-gcc-code

Reply:
Done.